src/utils/constants.ts

```diff
@@ -249,4 +249,24 @@ export const CONTRACT_ADDRESSES = {
     mainnet: '0xcA11bde05977b3631167028862bE2a173976CA11',
     calibration: '0xcA11bde05977b3631167028862bE2a173976CA11',
   } as const satisfies Record<FilecoinNetworkType, string>,
+
+  PAYMENTS: {
+    mainnet: '', // TODO
+    calibration: '0x1096025c9D6B29E12E2f04965F6E64d564Ce0750',
+  } as const satisfies Record<FilecoinNetworkType, string>,
+
+  PDP_VERIFIER: {
+    mainnet: '', // TODO
+    calibration: '0x445238Eca6c6aB8Dff1Aa6087d9c05734D22f137',
+  } as const satisfies Record<FilecoinNetworkType, string>,
+
+  SESSION_KEY_REGISTRY: {
+    mainnet: '', // TODO
+    calibration: '0x97Dd879F5a97A8c761B94746d7F5cfF50AAd4452',
+  } as const satisfies Record<FilecoinNetworkType, string>,
+
```

Comment:
let's rip these out, leave USDFC, I think it still compiles since you use MOCK_ADDRESSES for everything but USDFC

Reply:
a02e0a6